package.json

```diff
@@ -33,6 +33,7 @@
   "devDependencies": {
     "@11ty/eleventy": "^0.12.0",
     "@mozilla-protocol/tokens": "^5.0.5",
+    "@willdurand/addons-frontend-card": "^1.0.0",
```

Comment:
So the plan will be that this will be an internal package before this goes into production?

Reply:
I have a PR almoat ready for review in addons-frontend, which generates the same package.

That's one thing. The other is that we need to figure out how to release npm packages from addons-frontend. Maybw we need a special tag pattern to avoid stage deployments.

The `@willdurand` package is there to iterate on the blog without being blocked by the addons-frontend side.